tests/test_invenio_mail_tasks.py

```diff
@@ -64,3 +70,46 @@ def test_send_message_stream(email_task_app):
                 'From: test2@test2.test2') != -1
             assert result_stream.getvalue().find('To: test2@test2.test2') != -1
             assert result_stream.getvalue().find('Subject: Test2') != -1
+
+
+def test_send_message_with_attachments(email_task_app):
+    """Test sending a message with attachments."""
+    with email_task_app.app_context():
+        filename = pkg_resources.resource_filename(
+            __name__, os.path.join('attachments', 'invenio.svg'))
+        content_type = 'image/svg+xml'
+        data = pkg_resources.resource_string(
+            __name__, os.path.join('attachments', 'invenio.svg'))
```

Comment:
Looks ugly. Maybe I can hide it in a fixture? Or I'd love to have a solution which doesn't read from the FS.